File: tests/fixtures/openai-realtime/openai-realtime-verify-deprecated-session-fields-broken/02-agent-config.ts
export const REALTIME_WS_URL = 'wss://api.openai.com/v1/realtime?model=gpt-4o-realtime-preview';

function buildAgentConfigMsg(agentPrompt: string) {
  return {
    type: 'session.update',
    session: {
      modalities: ['text', 'audio'],
      instructions: agentPrompt,
      input_audio_format: 'g711_ulaw',
      output_audio_format: 'g711_ulaw',
      turn_detection: { type: 'server_vad' },
      temperature: 0.6,
    },
  };
}

export default buildAgentConfigMsg;

File: tests/fixtures/openai-realtime/openai-realtime-verify-deprecated-session-fields-broken/02_agent_config.py
import json


def build_agent_session_update():
    return json.dumps(
        {
            "type": "session.update",
            "session": {
                "voice": "alloy",
                "temperature": 0.8,
                "turn_detection": {"type": "server_vad"},
            },
        }
    )
